wrappers/rust/icicle-core/src/curve.rs

```diff
@@ -12,6 +12,11 @@ pub trait CurveConfig: PartialEq + Copy + Clone {
     fn eq_proj(point1: *const c_void, point2: *const c_void) -> c_uint;
```

Comment:
Since you introduced associated types to the trait (which is probably a good idea), `c_void` can be changed to `Projective` I think

Reply:
solved in ff2bf3e76ffeea86ec60824d2c5cdce3757d0f8f